lib/autoupdate.ps1

```diff
@@ -259,6 +264,10 @@ function get_hash_for_app([String] $app, $config, [String] $version, [String] $u
         $hashmode = 'sourceforge'
     }
 
+    if ($url -match 'https:\/\/github\.com\/(?<owner>[^\/]+)\/(?<repo>[^\/]+)\/releases\/download\/[^\/]+\/[^\/]+') {
```

Comment:
Suppose scoop should respect here explicitly specified checksums file as well (`!$hashfile_url -and ...`).

This will help to avoid automatic check errors and related frustration on updating manifests with "legacy" releases (created before automatic checksums generation for release assets was introduced).

See e.g. https://github.com/ScoopInstaller/Main/pull/7057#issuecomment-3176752012 ([log](https://github.com/ScoopInstaller/Main/actions/runs/16891017302/job/47850541846)):

```
   Searching hash for  
    curlie_1.8.2_windows_arm64.zip 
     in  
    https://github.com/rs/curlie/releases/download/v1.8.2/curlie_1.8.2_checksums.txt 
    Could not find hash in https://api.github.com/repos/rs/curlie/releases 
```

Reply:
Yeah...I have found this issue: https://github.com/ScoopInstaller/Scoop/pull/6434#issuecomment-3177400785

Reply:
Go to hotfix #6435 #6436